caf-audit/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditIndexManager.java

```diff
@@ -33,6 +41,8 @@
     private final LoadingCache<String, String> indexCache;
     private final ElasticAuditConfiguration config;
 
+    private static final String indexTypeName = "cafAuditEvent";
```

Comment:
Using the value in the constants file that holds the index type name in the latest commit.